content/applications/marketing/social_marketing/essentials/social_campaigns.rst

```diff
@@ -0,0 +1,139 @@
+==========================
+Social marketing campaigns
+==========================
+
+Social marketing campaigns help companies connect directly with the marketplace. These campaigns
+are vital when introducing a new product to customers, explaining the value of a service/product,
+or advertising an upcoming event or product release.
+
+The most effective social marketing campaigns involve multiple channels, so it's important to have
+a reliable solution to help with the planning, posting, tracking, and analyzing of content.
+
+Campaigns page
+==============
+
+To access a complete overview of all social marketing campaigns, open the :menuselection:`Social
+Marketing` application, and click :menuselection:`Campaigns` from the header menu. Doing so reveals
+a separate page with every campaign in a default kanban view.
+
+.. image:: social_campaigns/campaigns-page.png
+   :align: center
+   :alt: View of the campaigns page in the Odoo Social Marketing application.
+
+Each *stage* in the kanban view can be edited, by clicking the "gear" icon to the left of the "+"
+(plus sign) - located to the right of the stage title.
+
+The "gear" icon only appears when the cursor hovers to the left of the "+" (plus sign). When the
+"gear" icon is clicked, a drop-down menu reveals the options: *Fold*, *Edit Stage*, and *Delete*.
```

Comment:
I know there's some uncertainty around guilabel use and icons.
Imo it makes more sense to use them because we're instructing or hinting at the user to click them.
Ideally, I'd love to use emojis here (maybe one day).

Here's what I suggest, with that in mind, and also turning the second sentence into a **note**:
```suggestion
Each *stage* in the kanban view can be edited, by clicking the :guilabel:`gear icon` to the left of
the :guilabel:` + (plus sign icon)` - located to the right of the stage title.

.. note::
   The **gear icon** *only* appears when the cursor hovers to the left of the ** + (plus sign
   icon)**. When the gear icon is clicked, a drop-down menu reveals the options: :guilabel:`Fold`,
   :guilabel:`Edit Stage`, and :guilabel:`Delete`.
```